FirebaseStorage/Sources/FIRStorage.m

```diff
@@ -36,14 +36,59 @@
     NSMutableDictionary<NSString * /* bucket */, GTMSessionFetcherService *> *> *_fetcherServiceMap;
 static GTMSessionFetcherRetryBlock _retryWhenOffline;
 
-@interface FIRStorage () {
+//Should this be in FIRStorage_Private.h instead?
+@protocol FIRStorageInstanceProvider
```

Comment:
This doesn't need to be here since there's already the `FIRStorageComponent` class taking care of this.